broker-daemon/broker-rpc/admin-service/health-check.js

```diff
@@ -8,27 +8,32 @@ const STATUS_CODES = Object.freeze({
 })
 
 /**
- * Gets an engine status for a specified engine
+ * Returns an individual engine's status
  *
- * @param {Engine} engine
- * @return {String} OK
- * @return {String} error message if engine call fails
+ * @param {Array} req
+ * @param {String} req.symbol
+ * @param {Object} req.engine
+ * @return {Object} res
+ * @return {String} res.symbol - an engine's currency symbol (e.g. BTC or LTC)
+ * @return {String} res.status - an engine's status
  */
-async function getEngineStatus (engine) {
+async function getEngineStatus ([ symbol, engine ]) {
+  let status = STATUS_CODES.OK
```

Comment:
it would make the most sense to me to have `status` start off as `UNKNOWN` and only change to `OK` after checking that the engine `isAvailable`. I know the result is the same, but it looks weird this way.

Reply:
I might omit the default entirely just because an `UNKNOWN` status would never get hit with the current code